src/data/ws-templates.ts

```diff
@@ -0,0 +1,23 @@
+import { Connection, UnsubscribeFunc } from "home-assistant-js-websocket";
+
+interface RenderTemplateResult {
+  result: string;
+}
+
+export const subscribeRenderTemplate = (
+  conn: Connection,
+  template: string,
```

Comment:
Pass `template`, `entity_ids` and `variables` as 1 object that you explode into the message below.

`(conn, onChange, params) =>`

`{type: "render_template", ...params}`